### PR TITLE
feat: Plugin update for OpenEdx Quince

### PR DESCRIPTION
**Pull request by mhsiddiqui**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Here is a screenshot of the Jupyter XBlock in action:
 
 ## Usage
 
-Install this xblock with [Tutor](https://docs.tutor.overhang.io/) (Palm release):
+Install this xblock with [Tutor](https://docs.tutor.overhang.io/) (Quince release):
```
Comment (regisb) [inline README.md]: Can you please update the instructions to explain that this xblock is automatically installed by the tutor-jupyter plugin? We should refer users  to the plugin docs.

File: README.md [hunk 2]
```diff
@@ -7,19 +7,15 @@ Features:
 * Integrate [JupyterLab](https://jupyterlab.readthedocs.io/en/stable/) notebooks to the Open edX courseware.
 * Fully editable notebooks and student workspaces.
 * Simple integration of notebooks from public git repositories.
+* Works with OpenEdx Oreo, Palm, Quince
```
Comment (regisb) [inline README.md]: Oreo would have been a lovely name, but it was Olive :) Let's not add this line, it's unnecessary.

Comment (mhsiddiqui) [reply README.md]: Sorry. I confused Android releases with OpenEdx :)

Comment (mhsiddiqui) [reply README.md]: I have updated this.

**Review by regisb — APPROVED** [review]
Sweet! I'll merge this now and publish a new release such that we can build the quince image.

Edit: Done https://pypi.org/project/jupyter-xblock/17.0.0/

**Pull request merged**